test/e2e/bpmn.rendering.test.ts

```diff
@@ -129,6 +129,8 @@ describe('no visual regression', () => {
   ]);
 
   function getImageSnapshotConfig(fileName: string): jest.ImageSnapshotConfig {
+    let failureThreshold = 0.000003151;
```

Comment:
:question: I suggest we add a comment to explain where this value comes from (minimal value for local macOS builds for diagrams that render svg only?)

Reply:
Done